### DISCUSSION TITLE
How is tach pronounced?

### DISCUSSION THREAD

**Discussion by nip3o** [gauge-sh/tach#120] [General]
Every time I'm talking tach with my colleagues I'm struggling to figure out how it is supposed to be pronounced, and cannot find any information about it? 🤔

**Comment by emdoyle** [comment]
It is pronounced like 'tack' :)

It comes from the word [tachometer](https://en.wikipedia.org/wiki/Tachometer), also known as an RPM gauge. This is probably a good thing to add to the FAQ!
